cars.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
No, it must to be empty file.
Great work.

Reply:
Can I close this issue ?.

Reply:
No, I close it.